zest/releaser/release.py

```diff
@@ -57,6 +59,28 @@ def package_in_pypi(package):
         logger.debug("Package not found on pypi: %s", e)
         return False
 
+def _project_builder_runner(cmd, cwd=None, extra_environ=None):
+        """Call the subprocess, and format warnings and errors in red so that
+        they will work correctly with utils.show_interesting_lines()
```

Comment:
```suggestion
        """Run the build command and format warnings and errors.
        
        It runs the build command in a subprocess. Warnings and errors are formatted
        in red so that they will work correctly with utils.show_interesting_lines(). We 
        mimic the setuptools/wheels output that way.
```

Docstring formatting rule. One short line on the first line, then a blank line, then the rest.